packages/component-library/.storybook/preview.ts

```diff
@@ -18,6 +18,7 @@ import "@shopware-ag/meteor-tokens/administration/dark.css";
 
 const i18n = createI18n({
   // something vue-i18n options here ...
+  legacy: false,
```

Comment:
We should probably communitcate this change. Eventhough it will be released with the next major all the changes regarding Vue i18n need to be communicated. @Haberkamp @jleifeld

Reply:
Good thing. I will add it to the changelog!